trillian/ctfe/handlers.go

```diff
@@ -723,8 +723,10 @@ func getRoots(ctx context.Context, c *LogContext, w http.ResponseWriter, r *http
 	return http.StatusOK, nil
 }
 
-// See RFC 6962 Section 4.8. This is mostly used for debug purposes rather than by normal
-// CT clients.
+// See RFC 6962 Section 4.8. This is mostly used for debug purposes rather than
+// by normal CT clients.
+// TODO(pavelkalinnikov): This is actually very useful for computing hash of a
```

Comment:
Not sure the TODO is useful. Just remove the comment about debug? I'll be using it in DNS server as well.